models/issue.rb

```diff
@@ -137,6 +296,46 @@ def warning_message
       str.gsub(/\s+/, ' ')
    end
 
+   protected
```

Comment:
In Ruby, `public`, `protected`, and `private` set the visibility for all methods defined after the declaration, unless a method name is specified as a symbol argument (e.g., `protected :method_name`). So you only need one call until you want to stop making methods of the visibility that call sets.